app/src/androidTest/java/com/example/molliestephenson/tictactoe/MainActivityTest.java

```diff
@@ -0,0 +1,54 @@
+package com.example.molliestephenson.tictactoe;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.RootMatchers.withDecorView;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+@RunWith(AndroidJUnit4.class)
+public class MainActivityTest {
+
+    @Rule
+    public ActivityTestRule mActivityRule = new ActivityTestRule<>(MainActivity.class);
+
+    @Test
+    public void canWinAGame() {
+        onView(withId(R.id.left_top_corner)).perform(click());
```

Comment:
Is there a way to wrap `onView(withId(...))` in way that it makes sense in the context of a TTT game? Maybe we can write a wrapper function that gives it a great name?